### DISCUSSION TITLE
Check the rights inside the package

### DISCUSSION THREAD

**Discussion by kvinkinamoto** [spatie/laravel-permission#1941] [Q&A]
I have two projects. One in which I develop a package another in which I use this package. Where I develop and from not connected with git 
![composer dev](https://user-images.githubusercontent.com/1387047/143572399-c873f63e-2a63-4879-9182-597e4c0be53d.png)
check of permissions
       auth()->user()->cannot('cache_show_admin_menu') 
 works correctly. 

And in the project where it is connected from the git
![composer proj](https://user-images.githubusercontent.com/1387047/143572499-174a3b92-a6e7-4aa7-938e-a8598963b282.png)
, the rights check auth()->user()->cannot('cache_show_admin_menu')  always gives false

**Comment by erikn69** [comment]
Maybe you are forgetting the `guard_name`, don't expect many answers with so little information

**Reply by kvinkinamoto** [reply]
the only difference is the connection in the compositor
if you transfer the package to the project and connect as a type of path everything works. I found out that there was a problem with Laravel's guards. Spati test methods work approx. But the gates of Laravel do not tighten

**Reply by erikn69** [reply]
Maybe prioriy on Middlewares or ServiceProviders

**Reply by kvinkinamoto** [reply]
Try. No effect
